doc/reference/tooling/tt_cli/index.rst

```diff
@@ -7,15 +7,11 @@ tt CLI utility
 Tarantool-based applications. It covers a wide range of tasks -- from installing
 a specific Tarantool version to managing remote instances and developing applications.
 
-A multi-purpose tool for working with Tarantool from the command line, ``tt`` is
-a potential replacement for :ref:`tarantoolctl <tarantoolctl>`
-and :doc:`Cartridge CLI </book/cartridge/cartridge_cli/index>`.
-
 ``tt`` is developed in its own `GitHub repository <https://github.com/tarantool/tt>`_.
 Here you can find its source code, the changelog, and usage examples.
```

Comment:
> ... , and usage examples.

Given that documentation also contains usage examples (and, I believe, documentation is the main source of examples), this phrase confuses me a bit.

Reply:
Rephrased.